### ISSUE TITLE
Softmax Splatting for Video Frame Interpolation

### ISSUE THREAD

**Issue by KeisukeShimokawa** [KeisukeShimokawa/pytorch-papers-challenge#40]
### 論文へのリンク

[[arXiv:2003.05534] Softmax Splatting for Video Frame Interpolation](https://arxiv.org/abs/2003.05534)

### 著者・所属機関

Simon Niklaus, Feng Liu

- Portland State University

### 投稿日時（YYYY-MM-DD）

2020-03-11

**Comment by KeisukeShimokawa** [comment]
[web page](http://sniklaus.com/papers/softsplat)
[https://github.com/sniklaus/softmax-splatting](https://github.com/sniklaus/softmax-splatting)
